### PR TITLE
Make the list item responsive

### PR DESCRIPTION
**Pull request by m6jones**
image

### CODE REVIEW

File: components/d2l-activity-list-item/d2l-activity-list-item.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+import {beforeNextRender, afterNextRender} from '@polymer/polymer/lib/utils/render-status.js';
+import {IronResizableBehavior} from '@polymer/iron-resizable-behavior/iron-resizable-behavior.js';
+import 'fastdom/fastdom.min.js';
 import '../../../d2l-offscreen/d2l-offscreen-shared-styles.js';
```
Comment (qwangd2l) [inline components/d2l-activity-list-item/d2l-activity-list-item.js]: Not sure if it's a big deal but the recommended way to import external packages is just with the package name. (Dave L's post in #polymer-3-upgrade)

File: components/d2l-activity-list-item/d2l-activity-list-item.js [hunk 2]
```diff
@@ -105,28 +117,162 @@ class D2lActivityListItem extends PolymerElement {
...
 		return {
 			href: String,
 			_text: String,
 			_imageUrl: String,
 			_title: String,
-			_description: String,
-			_tags: String
+			_description: {
+				type: String,
+				observe: '_onDescriptionChange'
```
Comment (qwangd2l) [inline components/d2l-activity-list-item/d2l-activity-list-item.js]: is this supposed to be `observer`?

File: package.json [hunk 3]
```diff
@@ -45,6 +45,7 @@
 		"d2l-icons": "BrightspaceUI/icons#semver:^6.0.0",
 		"d2l-offscreen": "BrightspaceUI/offscreen#semver:^4.0.0",
 		"d2l-polymer-behaviors": "Brightspace/d2l-polymer-behaviors-ui#semver:^2.0.0",
-		"d2l-typography": "BrightspaceUI/typography#semver:^7.0.0"
+    "d2l-typography": "BrightspaceUI/typography#semver:^7.0.0",
```
Comment (AdmiralCoco) [inline package.json]: Not sure if you've run into this but I've seen some of the path issues described in `#polymer-3-upgrade`; I resolved them by updating all relevant packages: 

```js
{
		"d2l-icons": "BrightspaceUI/icons#semver:^6.0.1",
		"d2l-offscreen": "BrightspaceUI/offscreen#semver:^4.0.1",
		"d2l-polymer-behaviors": "Brightspace/d2l-polymer-behaviors-ui#semver:^2.0.1",
		"d2l-typography": "BrightspaceUI/typography#semver:^7.0.2"	   //(<- updating now)
}
```

Comment (m6jones) [reply package.json]: I haven't but I will update it now and read the channel again.

File: demo/d2l-activity-list-item/index.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+						item2._imageUrl = 'ImageUrl';
+						item2._title = 'Financial Planning';
+						item2._description = 'Financial Planning & Analysis: Building a Company\'s Budget from Scratch guides you through the core principles of Financial Planning and helps you to understand how a company prepares its budget. The video lessons touch on several awesome. Ultra-Financial Planning & Analysis  in the 20th Century: Building a Successful Budget';
+						item2._tags = ['Duration 1h30m', 'Metadata', 'Metadata'];
+					</script>
+				</template>
+			</demo-snippet>
+		</div>
+	</body>
+	<!-- The class="d2l-typography" on the body doesn't always work -->
```
Comment (qwangd2l) [inline demo/d2l-activity-list-item/index.html]: Is this the case still?
Have you tried adding `unresolved class="d2l-typography"` to the body tag?

Comment (m6jones) [reply demo/d2l-activity-list-item/index.html]: I didn't actually try it. It is no longer required! Sweet.

**Pull request merged**